packages/grafana-ui/src/components/Tags/Tag.tsx

```diff
@@ -51,6 +56,7 @@ const getTagStyles = (theme: GrafanaTheme, name: string, colorIndex?: number) =>
   }
   return {
     wrapper: css`
+      border-style: none;
```

Comment:
you might want `appearance: none` also to remove some other button styling that (mobile safari?) likes to include.